content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,213 @@
+Generally, when you use a FaaS (Function as a Service) environment to interact with a database, every function invocation can result in a new connection to the database. This is not a problem with a constantly running Node.js server. Therefore, it is beneficial to pool DB connections to get better performance. To solve this issue, you can use the [Data Proxy](/data-platform/data-proxy). For other solutions, see the [connection management guide for serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
 
-> **Note:** It's best practice to keep secret information, such as `DATABASE_URL`, out of your git repository. To do this, add your `.env` file to `.gitignore`. This guide copies the source and does not create a repository, so you only need to update `.gitignore` if you initialize a repository.
+## Deploy only the required files
 
-## 3. Push the Prisma schema to the database
+To reduce your deployment footprint, you can update your deployment process to only upload the files your application needs. The Serverless configuration file `serverless.yml` below shows a `package` pattern that includes only the Prisma engine file relevant to the Lambda runtime and excludes the others. This means that when Serverless Framework packages your app for upload, it includes only one engine file. This ensures the packaged archive is as small as possible.
 
-`prisma db push` takes the models and attributes that you define in `schema.prisma` and pushes them to the database.
+```code file=serverless.yml no-copy
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
+```
 
-### Steps
+If you are deploying to [Lambda functions with ARM64 architecture](#lambda-functions-with-arm64-architectures) you should update the Serverless configuration file to package the `arm64` engine file, as follows:
 
-```terminal
-npx prisma db push
+```code file=serverless.yml highlight=4;normal
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-linux-arm64-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
 ```
 
-This command also generates Prisma Client based on your Prisma schema.
+If you use `serverless-webpack`, see [Deployment with serverless webpack](#deployment-with-serverless-webpack) below.
 
-### Result
+## Deployment with <inlinecode>serverless-webpack</inlinecode>
 
-Prisma applies the models that are defined in `schema.prisma` to the database.
+If you use `serverless-webpack`, then you will need additional configuration so that your `schema.prisma` is properly bundled. You will need to:
```

Comment:
```suggestion
If you use `serverless-webpack`, you will need additional configuration so that your `schema.prisma` is properly bundled. You will need to:
```